data/nav.yml

```diff
@@ -557,7 +557,11 @@
             - name: "Members"
               path: "apis/rest-api/organizations/members"
               pill: "beta"
-        - name: "Pipelines "
```

Comment:
Removed extra space.

Reply:
No this needs to be there unfortunately for the reason I've now explained in a comment within the relevant areas of the `nav.yml` file.